### PR TITLE
setup basic styles for landing page footer

### LINKED ISSUE #58: [Landing Page] Footer
image

Add a responsive footer to the landing page.
- The footer should have similar behavior to the footer on [morehumaninternet.org](https://morehumaninternet.org/).
- The GitHub icon should link to the [extension repo](https://github.com/morehumaninternet/roar-extension).
- The Donate button should link to our [buymeacoffee account](https://www.buymeacoffee.com/morehumaninter).

### PR DESCRIPTION
**Pull request by hankeliu2015**
basic styles for footer. Work in progress

### CODE REVIEW

File: public/index.css [hunk 1]
```diff
@@ -202,3 +202,57 @@ body {
...
   }
 }
+
+/* footer styles  */
+
+.mhi-footer {
+  /* display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between; */
```
Comment (shachar-langer) [inline public/index.css]: Please remove

File: public/index.css [hunk 2]
```diff
@@ -202,3 +202,57 @@ body {
...
+  font-size: 0.77777em;
```
Comment (shachar-langer) [inline public/index.css]: This is a very specific font size. Let's round it up to 0.8

File: public/index.css [hunk 3]
```diff
@@ -202,3 +202,57 @@ body {
...
+  padding-left: 25px;
+  padding-right: 25px;
+  /* background-color: #f9f9f9; */
+  padding-top: 2.5rem;
```
Comment (shachar-langer) [inline public/index.css]: We want to footer width to be 100% of the screen size so padding-right and padding-left are redundant.

File: public/index.css [hunk 4]
```diff
@@ -202,3 +202,57 @@ body {
...
+}
+
+.footer-header {
```
Comment (shachar-langer) [inline public/index.css]: The naming `footer-header` is... confusing. The text "made with... " isn't really a header

File: public/index.css [hunk 5]
```diff
@@ -202,3 +202,57 @@ body {
...
+.footer-header > h4 > svg {
+  transform: translate(0, 25%);
+}
+.mhi-logo {
+  position: relative;
+  /* padding: 0.5rem 0; */
+  width: 135px;
+  left: 50%;
+  transform: translate(-50%);
+}
```
Comment (shachar-langer) [inline public/index.css]: Centering text on a page is a problem with many solutions. This solution works but I believe there are better solutions. We can get rid of the `position`, `width`, `left`, and `transform` attributes by wrapping the "header" and the MHI logo in a div and replacing all the properties here with a flexbox:
```
.new-div {
  display: flex;
  flex-direction: column;
  align-items: center;
}
```

File: public/index.css [hunk 6]
```diff
@@ -202,3 +202,57 @@ body {
...
+  background-color: var(--human-blue);
+  width: 100%;
+  margin-top: 5rem;
+  height: 3.5rem;
+  padding: 0 1rem;
+}
+
+.footer-links {
+  min-width: 160px;
+}
```
Comment (shachar-langer) [inline public/index.css]: Can you please explain why is this necessary?

File: public/index.css [hunk 7]
```diff
@@ -202,3 +202,57 @@ body {
...
+.footer-banner p {
+  /* display: inline; */
+  /* float: right; */
+  /* min-width: 333px; */
```
Comment (shachar-langer) [inline public/index.css]: Please remove redundant code

File: public/index.html [hunk 8]
```diff
@@ -155,7 +155,45 @@ <h3 class="acc-header">
   </main>
   </div>
 
-  
+  <footer class="mhi-footer">
```
Comment (shachar-langer) [inline public/index.html:158]: There are a couple `div`s that we don't really need here. Let's try to optimize the div count for the minimum required.

File: public/index.html [hunk 9]
```diff
@@ -155,7 +155,45 @@ <h3 class="acc-header">
...
+        </svg>
+      </a>  
+    </div>
+    <div class="footer-banner">
+      <div class="footer-links">
+        <a class="footer-link" href="https://github.com/morehumaninternet/frontend" aria-label="Github"><svg width="25" height="25" viewBox="0 0 31 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (shachar-langer) [inline public/index.html]: All the links work great. Can you please open them in a new tab?

File: public/index.css (hunk 6 above)
Comment (hankeliu2015) [reply public/index.css]: wanted to prevent the footer links wraps like this. If we not mind the wrap,  I will removed it. Thanks!
Screen Shot 2021-01-01 at 12 11 10 PM

File: public/index.css (hunk 3 above)
Comment (hankeliu2015) [reply public/index.css]: The left and right was too close the the border before wrapping the links elements into a div. Removed.

File: public/index.css (hunk 2 above)
Comment (hankeliu2015) [reply public/index.css]: simply followed the frontend home page footers styles. corrected.

File: public/index.css (hunk 7 above)
Comment (hankeliu2015) [reply public/index.css]: removed

File: public/index.html (hunk 9 above)
Comment (hankeliu2015) [reply public/index.html]: added new tabs for all footer's links.

File: public/index.css (hunk 6 above)
Comment (hankeliu2015) [reply public/index.css]: decided to keep the mini-width, where icons will stay straight on smaller screen.

File: public/index.css (hunk 5 above)
Comment (hankeliu2015) [reply public/index.css]: changed as suggested

File: public/index.css (hunk 4 above)
Comment (hankeliu2015) [reply public/index.css]: removed footer-header class, merged the elements into another div

File: public/index.css (hunk 1 above)
Comment (hankeliu2015) [reply public/index.css]: removed

File: public/index.html (hunk 8 above)
Comment (hankeliu2015) [reply public/index.html:158]: removed extra divs as suggested

File: public/index.html [hunk 10]
```diff
@@ -155,7 +155,44 @@ <h3 class="acc-header">
...
-  
+  <footer class="mhi-footer">
+    
+    <div class="mhi-logo">
+      <h4>Built with<svg width="33" height="17"  viewBox="0 0 20 19" fill="none">
+        <path d="M14.5 0C11.605 0 10 2.09 10 2.09C10 2.09 8.395 0 5.5 0C2.462 0 0 2.462 0 5.5C0 9.671 4.912 13.713 6.281 14.99C7.858 16.46 10 18.35 10 18.35C10 18.35 12.142 16.46 13.719 14.99C15.088 13.713 20 9.671 20 5.5C20 2.462 17.538 0 14.5 0Z" fill="#FA759E" />
+      </svg>by the team at</h4>
+
+      <a target="_blank" href="https://morehumaninternet.org/">
+        <svg width="133" height="55" viewBox="0 0 133 55" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (will-weiss) [inline public/index.html]: This svg is scaled incorrectly. You can see that the `ellipse`s have a different rx than ry which makes them appear skewed. Please use the [svg used in the roar-extension](https://github.com/morehumaninternet/roar-extension/blob/e603db07123cbe91ec7fdab05ade5726ec7eb642/src/popup/views/not-authed.tsx#L20)

File: public/index.css [hunk 11]
```diff
@@ -202,3 +202,41 @@ body {
...
+
+.mhi-logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: var(--human-blue);
+}
+
+.mhi-logo > h4 > svg {
+  transform: translate(0, 25%);
```
Comment (will-weiss) [inline public/index.css]: What is this doing?

File: public/index.css [hunk 12]
```diff
@@ -202,3 +202,41 @@ body {
...
+  flex-wrap: wrap;
+  background-color: var(--human-blue);
+  width: 100%;
+  margin-top: 5rem;
+  height: 3.5rem;
+  padding: 0 1rem;
+}
+
+.footer-links {
+  min-width: 160px;
```
Comment (will-weiss) [inline public/index.css]: Magic numbers like this will break very easily. Let's eliminate this.

File: public/index.html [hunk 13]
```diff
@@ -155,7 +155,44 @@ <h3 class="acc-header">
   </main>
   </div>
 
-
```
Comment (will-weiss) [inline public/index.html:158]: Rather than applying a `fill="#E9E9E9" on each svg, let's do that with CSS so we can also change the colors on hover

```css
.footer-link > svg {
  fill: #e9e9e9;
}
.footer-link:hover > svg {
  fill: white;
}

**Review by will-weiss — CHANGES_REQUESTED** [review]

**Comment by will-weiss** [comment]
@hankeliu2015 Thanks for your work on this, I pushed a few changes which mainly address the behavior on mobile. We didn't quite specify, but I think a two row centered layout looks nice on mobile.

Left a few comments for a few other things that need fixing, then let's get this merged!

image

File: public/index.css (hunk 11 above)
Comment (hankeliu2015) [reply public/index.css]: offset the heart position. 
Screen Shot 2021-01-01 at 3 34 45 PM
Screen Shot 2021-01-01 at 3 35 08 PM

File: public/index.css (hunk 12 above)
Comment (hankeliu2015) [reply public/index.css]: remove it.

File: public/index.html (hunk 13 above)
Comment (hankeliu2015) [reply public/index.html:158]: changed as suggested

File: public/index.css (hunk 11 above)
Comment (hankeliu2015) [reply public/index.css]: style the heart logo as discussed

File: public/index.html (hunk 10 above)
Comment (hankeliu2015) [reply public/index.html]: replaced svg file and styles as suggested

**Review by will-weiss — APPROVED** [review]
